modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7513,6 +7513,8 @@ feature.flag.LPD-31212.description=Users can create rich text content that inclu
 feature.flag.LPD-31212.title=Disable XSS Validations in Rich Text Editor
 feature.flag.LPD-31789.description=Allow administrators to configure Google Vertex AI as text embedding provider for semantic search.
 feature.flag.LPD-31789.title=Google Vertex AI Text Embedding Provider
+feature.flag.LPD-34650.description=Developers can apply theme CSS and global CSS client extensions to an entire instance.
```

Comment:
About the title, what do you think about using the ticket title?
About the description, what do you think about using "Admin interface/page" instead of "entire instance"? Just to keep the same business domain context

Reply:
@algobob  We have another feature flag similar to this one.

```
feature.flag.LPD-30371.description=Developers can apply JavaScript client extensions to an entire instance.
feature.flag.LPD-30371.title=Instance Global JavaScript Client Extensions
```

Since the Global JS one was already approved by Brian, I kept it the same.

Even the "to an entire instance" was added by Brian. I find it weird, I prefer "to an instance"